Style-guide/README.md

```diff
@@ -81,7 +78,7 @@ Descriptive and consistent naming makes software easier to read and understand.
 - naming methods for their side effects
   - verb methods follow the -ed, -ing rule for the non-mutating version
   - noun methods follow the formX rule for the mutating version
-  - boolean types should read like assertions
+  - boolean types should read like assertions (`isLoading`, `isHidden`, etc)
   - protocols that describe _what something is_ should read as nouns
```

Comment:
We currently have many violations of this, eg, `SomethingProtocol`

Reply:
Our frequent use of the `SomethingProtocol` is a side effect of wrapping our business controllers in protocols. I am not sure how we should change this. To have `PaymentsController` conform to `PaymentsControlling` instead of `PaymentsControllerProtocol` does not sound right to me. Perhaps the underlying problem is that we should model business controllers as protocols.

Reply:
> If an associated type is so tightly bound to its protocol constraint that the protocol name is the role, avoid collision by appending Protocol to the protocol name

Swift Guidelines say that at times it's "kinda ok".